src/test/java/codeu/controller/ProfileServletTest.java

```diff
@@ -0,0 +1,47 @@
+package codeu.controller;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+public class ProfileServletTest {
+
+  private ProfileServlet profileServlet;
+  private HttpServletRequest mockRequest;
+  private HttpServletResponse mockResponse;
+  private RequestDispatcher mockRequestDispatcher;
+  private UserStore mockUserStore;
+
+  @Before
+  public void setup() {
+    profileServlet = new ProfileServlet();
+    mockRequest = Mockito.mock(HttpServletRequest.class);
+    mockResponse = Mockito.mock(HttpServletResponse.class);
+    mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
+    Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/profile.jsp"))
+        .thenReturn(mockRequestDispatcher);
+    mockUserStore = Mockito.mock(UserStore.class);
+    profileServlet.setUserStore(mockUserStore);
+  }
+
+  @Test
+  public void testDoGet() throws IOException, ServletException {
+    profileServlet.doGet(mockRequest, mockResponse);
+
+    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
+  }
+
```

Comment:
Why do you not have a test for doPost? It's a pretty simple function but might as well stick a quick test for it in there.

Reply:
Good point! It would be helpful to have tests for each main function even it's simple for now, since we might need to add more code after.